lib/main_common.pm

```diff
@@ -1600,7 +1600,7 @@ sub load_extra_tests_console {
     loadtest 'console/procps';
     loadtest "console/lshw" if ((is_sle('15+') && (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'x86_64'))) || is_opensuse);
     loadtest 'console/quota'   unless is_jeos;
-    loadtest 'console/zziplib' unless is_jeos;
+    loadtest 'console/zziplib' if is_sle('12-SP3+');
```

Comment:
Looks like `is_jeos` condition was removed by accident. Please extend condition to don't run it on JeOS.

Reply:
done!